crates/bevy_hierarchy/src/valid_parent_check_plugin.rs

```diff
@@ -3,8 +3,9 @@ use std::marker::PhantomData;
 #[cfg(feature = "bevy_app")]
 use crate::Parent;
 use bevy_ecs::prelude::*;
+use bevy_reflect::ShortName;
```

Comment:
```suggestion
#[cfg(feature = "bevy_app")]
use bevy_reflect::ShortName;
```
Shouldn't this still be behind the feature?